transformer/classes/dockerfile/java/mavenanalyser.go

```diff
@@ -210,6 +210,33 @@ func (t *MavenAnalyser) Transform(newArtifacts []transformertypes.Artifact, oldA
 		if len(selectedMavenProfiles) == 0 {
 			logrus.Debugf("No maven profiles selected")
 		}
+
+		classifier := ""
+		// Iterate over existing plugins
+		for _, mavenPlugin := range *pom.Build.Plugins {
+			// Check if spring-boot-maven-plugin is present
+			if mavenPlugin.ArtifactID == "spring-boot-maven-plugin" {
+				for _, mavenPluginExecution := range *mavenPlugin.Executions {
```

Comment:
Since it is pointer we should probably check if executions is empty

Reply:
done